radio/src/targets/common/arm/stm32/adc_driver.cpp

```diff
@@ -52,6 +52,17 @@
   const int8_t adcDirection[NUM_ANALOGS] = {-1,1,-1,1,  1,1,  1};
 #elif defined(PCBXLITE)
   const int8_t adcDirection[NUM_ANALOGS] = {1,-1,-1,1,  -1,1,  1,  1};
+#elif defined(PCBNV14)
```

Comment:
Now that the NV14 is using it's own driver, please remove the changes here.

Reply:
Reverted to original driver.